### PR TITLE
Translation french

### PR DESCRIPTION
**Pull request by le-jun**
Gave a shot to a french translation.

### CODE REVIEW

**Comment by Robbendebiene** [comment]
Thanks a lot for the translation!
There is [one more file that translates the App UI](https://github.com/OPENER-next/OpenStop/blob/master/lib/l10n/app_en.arb). Just in case you missed it and want to translate it 🙂

**Comment by le-jun** [comment]
Oh right, missed it. Give me a week or so and I'll add it to this PR.

File: lib/l10n/app_fr.arb [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+  "and": "et",
+  "@and": {},
+  "more": "plus",
+  "@more": {},
+  "element": "élément",
+  "@element": {},
+
+  "durationInputDaysLabel": "Jours",
+  "@durationInputDaysLabel": {},
+  "durationInputHoursLabel": "Heuress",
```
Comment (Robbendebiene) [inline lib/l10n/app_fr.arb:241]: I think there is one "s" too many.

Comment (le-jun) [reply lib/l10n/app_fr.arb]: Indeed

**Comment by Robbendebiene** [comment]
Thanks again for all the work!

**Pull request merged**